docs/guides/manage_keys.md

```diff
@@ -17,21 +18,21 @@ inputset = range(10)
 circuit = f.compile(inputset)
 ```
 
-Circuits have a property called `keys` of type `fhe.Keys`, which has several utility functions dedicated to key management!
+Circuits have a `keys` property of type `fhe.Keys`, which includes several utilities for key management.
 
 ## Generation
 
-To explicitly generate keys for a circuit, you can use:
+To explicitly generate keys for a circuit, use:
 
 ```python
 circuit.keys.generate()
 ```
 
 {% hint style="info" %}
-Generated keys are stored in memory upon generation, unencrypted.
+Generated keys are stored in memory and remain unencrypted.
 {% endhint %}
 
-And it's possible to set a custom seed for reproducibility:
+You can also set a custom seed for reproducibility:
```

Comment:
I am surprised we have this, it's really dangerous, security wise, @BourgerieQuentin @umut-sahin

Reply:
Maybe we can raise a warning about it being unsafe, wdyt?

Reply:
Yes you are true that's not safe, if the seed do not come from a crypto secure generator, but I think is important to let this features, else for reproductability, else because a power user would want to generate itself crypto secure seed.
Maybe we could just prefix `seed` and `encryption_seed` by `unsafe_`

Reply:
And not safe in the sense of: 
- if the entropy of 'seed' is too small, we'll not generate sufficiently random keys, it's really bad
- if a user reuse the same seed, the same key is generated, it's really bad

What happens if the seed is not set? Do we really have sufficient entropy for the key?

Reply:
Yes as well if the seed is not set we will use tfhe-rs seed generators (using rdseed on linux if available). So using custom seed is only for power user or for debugging purpose.

Reply:
will be done in https://github.com/zama-ai/concrete-internal/issues/840, let's merge this PR to not make Yuxi wait